lib/pure/uri.nim

```diff
@@ -380,6 +380,57 @@ proc `$`*(u: Uri): string =
     result.add("#")
     result.add(u.anchor)
 
+type
+  Query* = distinct Table[string, seq[string]]
```

Comment:
A table can easily hold multiple values for the same key, no ``seq[string]`` required.